README.md

```diff
@@ -65,14 +65,14 @@ Recommended:
 
 * Fast CPU with 4+ cores
 * 16GB+ RAM
-* High Performance SSD with at least 1TB free space
+* High-Performance SSD with at least 1TB of free space
```

Comment:
```suggestion
* High-Performance SSD with at least 1TB of free space
```
Looks odd to me with the capitalization of `Performance`. It was there from the beginning, but was wrong there too, so please remove that

Reply:
Done.